lib/ruby_lsp/store.rb

```diff
@@ -37,13 +37,18 @@ def get(uri)
       return document unless document.nil?
 
       path = T.must(uri.to_standardized_path)
-      set(uri: uri, source: File.binread(path), version: 0)
+      language_id = path.end_with?(".erb") ? "erb" : "ruby"
```

Comment:
TIL `rhtml`. I think we'll need to add it to the extension's `documentSelector` too?

Reply:
Not necessary because the grammar already assigns the `erb` language ID to that file type. Similar to how we assign the `ruby` ID to several different files, like `Rakefile`, `Gemfile`, etc.